docs/guides/syntax-highlighting.server.mdx

```diff
@@ -1,158 +1,204 @@
-# Syntax highlighting
-
-There are two primary approaches for adding syntax highlighting to MDX:
+export const navSortSelf = 4
 
-*   composition via the MDXProvider
-*   remark plugin
+# Syntax highlighting
 
-It’s typically preferred to take the compositional approach, but both
-will be documented here.
+This guide explores how to apply syntax highlighting to code blocks.
 
-## Composition
+MDX supports standard markdown syntax ([CommonMark][]).
+It does not apply syntax highlighting to code blocks by default.
+There are two ways to accomplish syntax highlighting: at compile time or at
+runtime.
+Doing it at compile time means much less code is sent down the wire (syntax
+highlighting needs a *lot* of code).
+Doing it at runtime gives some flexibility but depends on what framework you use
+(so it’s specific to react, preact, vue, etc.) and moves all the work to your
+users.
```

Comment:
same for highlighting, styles are not included by default, a CSS file needs to be included somewhere

Reply:
It’s mentioned after the example (L38)